install/envs/mac.yml

```diff
@@ -30,16 +29,17 @@ dependencies:
   - torchvision
   - torchaudio
   - pytorch-lightning
-  - numpy
   - psutil
   - kivy=2.0.0
   - plotly
   - pyyaml
   - fastai
   - pip:
-    - tensorflow==2.2.0
+    - --no-binary :h5py: h5py==3.1.0
+    - tensorflow==2.5.3
     - git+https://github.com/autorope/keras-vis.git
     - simple-pid
-    - opencv-python-headless
     - moviepy
+    - install -U --no-deps numpy==1.19.4
```

Comment:
I think this does not work.

Reply:
fixed this to make it a legal set of arguments for `pip install`